lib/AllProductScreen.dart

```diff
@@ -0,0 +1,53 @@
+// ignore_for_file: prefer_const_constructors, file_names
```

Comment:
do not ignore file_names and prefer_use_const 

 - file name must not capital letters that's makes your code is neat please read the conventions in the boilerplate  readme file 
 - prefer use const , pls read why const and final keywords important for performance